static/js/drafts.js

```diff
@@ -128,12 +128,20 @@ export function confirm_delete_all_drafts() {
     });
 }
 
-export function rename_topic(stream_id, old_topic, new_topic) {
+export function rename_stream_recipient(old_stream_id, new_stream_id, old_topic, new_topic) {
```

Comment:
I feel like `old_stream_id, old_topic, new_stream_id, new_topic` would be a more intuitive parameter order, so you're describing (old location, new location)?

Reply:
yes I agree!